### ISSUE TITLE
Invalid System Image

### ISSUE THREAD

**Issue by Kainzor** [xamarin/get-started#89]
Problem encountered on https://dotnet.microsoft.com/learn/xamarin/hello-world-tutorial/devicesetup#
Operating System: windows

I'm basically just trying to use the Android Emulator but once I start it I get
![Screenshot 2021-05-25 131055](https://user-images.githubusercontent.com/23722261/119503622-b4070f80-bd5a-11eb-977b-3e16bd15d1aa.png)

I click the reinstall it unzips and does the whole process and when I try run again it gives out the same error
Basically just stuck in a loop

**Comment by jamesmontemagno** [comment]
Checkout the docs on emulators: https://docs.microsoft.com/en-us/xamarin/android/get-started/installation/android-emulator/
